src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -869,6 +875,12 @@ public static <T extends Member> T reflectAs(Class<T> expected, MethodHandle tar
      * it cannot be used to obtain another private {@code Lookup} by calling
      * {@link MethodHandles#privateLookupIn(Class, Lookup) privateLookupIn}
      * because it has no {@code MODULE} access.
+     * <p>
+     * The {@code Lookup} object returned by {@code privateLookupIn} has the access to
+     * {@linkplain Lookup#defineClass(byte[]) inject classes} in the runtime package
+     * of {@code T}. Extreme caution should be taken when opening a package
+     * to another module.  The injected classes have the same full privilege
+     * access as other members in the target module.
```

Comment:
```suggestion
     * <p>
     * The {@code Lookup} object returned by {@code privateLookupIn} is allowed to
     * {@linkplain Lookup#defineClass(byte[]) define classes} in the runtime package
     * of {@code T}. Extreme caution should be taken when opening a package
     * to another module as such defined classes have the same full privilege
     * access as other members in the target module.
```

You mention "target module" but i don't think i it is defined for the Lookup class JavaDoc. In this case are we referring to module M2?

Reply:
yes, M2.  Updated.